miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js

```diff
@@ -989,7 +994,12 @@ var Options = Options || {
   },
 
   confirmDelete: function (endpoint) {
+	var pieces = endpoint.split('/');
+	var singular = pieces[pieces.length - 2];
+	localStorage['tablediv'] = singular + 's';
     if (endpoint.indexOf("sampleclass") != -1) {
+      // need a modified tablediv because of pluralization
```

Comment:
You make a good point. I'll change the table div #ids to be the same as the singular resource which will avoid all pluralization messiness.